### DISCUSSION TITLE
ViewModifier.on Convenience Function

### DISCUSSION THREAD

**Discussion by gili-labs** [nalexn/ViewInspector#72] [Ideas]
The View.on convenience function saves us from a great deal of boilerplate. Is is possible to provide a similar convenience function for view modifiers conforming to Inspectable?

**Comment by gili-labs** [comment]
Alexey, I implemented this convenience function, tested it, and submitted a pull request.

**Reply by nalexn** [reply]
That's awesome, thank you @gili-labs ! I've released a new version 0.6.2 with this pull request. Also, thanks for becoming a sponsor! As promised, your account is now listed on my blog 👍 (should appear after Github pages finish the deploy)
